keyboards/tyxos63/matrix.c

```diff
@@ -0,0 +1,284 @@
+/*
```

Comment:
Please convert this to matrix lite. 

https://docs.qmk.fm/#/custom_matrix?id=lite